.github/workflows/ci.yaml

```diff
@@ -0,0 +1,125 @@
+name: CI
+
+on: [push]
+
+jobs:
+  cvc4:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Inspect
+      run: |
+         echo $PWD
+         find .
+    - name: Get CVC4
+      run: |
+        git clone -b eqrange-quant --single-branch https://github.com/GaloisInc/vadd-CVC4.git .
+    - name: Get Antlr
+      run: ./contrib/get-antlr-3.4
+    - name: Configure
+      run: ./configure.sh production --static --proofs --prefix=$PWD/dist
+    - name: Build
+      working-directory: build
+      run: make -j 8
+    - name: Package
+      run: |
+        find build -name cvc4
+        readelf -d build/bin/cvc4
+    - name: cvc4 artifact
+      uses: actions/upload-artifact@v1
+      with:
+        name: cvc4
+        path: build/bin/cvc4
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        # NOTE: must list all targets in `include` here to activate them
+        name: ["Linux 8.8.3"]
+        include:
+          - name: Linux 8.8.3
+            ghc-ver: 8.8.3
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+      with:
+        submodules: true
+    - name: Get GHC
+      run: |
+        sudo add-apt-repository ppa:hvr/ghc
+        sudo apt-get update
+        sudo apt-get install cabal-install-3.0 ghc-${{ matrix.ghc-ver }} llvm
+    - name: Cache
+      uses: actions/cache@v1
+      with:
+        path: home/.cabal/store/ghc-${{ matrix.ghc-ver }}
```

Comment:
This path is not equivalent to `path: ~/.cabal/store/ghc-${{ matrix.ghc-ver }}` which is the correct one.

Reply:
It's intentionally not equivalent.  I think caching requires the directory is a subdirectory of the repo.

Reply:
Ah, I see that now. The caching behavior used to be documented as needing to be a subdirectory, but now the documentation has been clarified to say that it can be either absolute path or relative to the repo.

[The official examples](https://github.com/actions/cache/blob/master/examples.md#haskell---cabal) show how they cache `~/.cabal/store`